static/downloadables/parse_associations.py

```diff
@@ -475,6 +475,9 @@ def parse_vcf(filteredFilePath, clumpsObjDict, tableObjDict, possibleAlleles, sn
                                                         index_snp_map[sample][clumpNum] = rsID, riskAllele, alleles if complements is None else complements
                                                         clumpedVariants.add(index_snp)
                                                     else:
+							# TODO: What if the current SNP also has two missing alleles? In this case, wouldn't we
```

Comment:
@mpage21  @MattCloward Can you guys take a look at this TODO comment? I think I may have found something that needs to change in the clumping logic, but want to make sure I'm not going crazy.

Reply:
I think that is a valid change to be made.